docs/tutorial/1-serialization.md

```diff
@@ -275,20 +275,22 @@ We'll also need a view which corresponds to an individual snippet, and can be us
 
 Finally we need to wire these views up.  Create the `snippets/urls.py` file:
 
-    from django.conf.urls import url
+    from django.urls import path
+    # note that the path function was added in Django 2.0
```

Comment:
I'd suggest we drop these two lines of comment.

Reply:
Will do, I'll try to get through the rest of the tutorial pages in the next two days and update them to the new routing syntax as needed.